Cockpit 333 🥃 (#787)
cockpit-project/cockpit-project.github.io: _posts/2025-02-13-cockpit-333.md

---
title: Cockpit 333
author:
date: '2025-02-13'
tags: podman, files
slug: cockpit-333
category: release

> summary: ''

Reviewer: ```suggestion
summary: 'Podman service is automatically started on demand, pasted files can have a different ownership'
```

---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 333 🥃, cockpit-podman 101, and cockpit-files 16:

> ## Podman: Automatically start podman.socket

Reviewer 2: This needs to go. This was already released in version 100 in https://cockpit-project.org/blog/cockpit-332.html , someone just forgot to drop the release-note label :cry:

Author: Oops.  Dropped.

If the system or user's `podman.socket` unit isn't running, cockpit-podman now silently starts it on its own, instead of asking the user about it. There is no need to have the service enabled (i.e. always start it on boot or user login) for cockpit-podman any more.

> ## Files: Paste as administrator
> 
> Files now allow administrators to change the permission of files when copy/pasting into directories owned by different users.

Reviewer: Coffee is slowly taking effect; apologies for not realizing this earlier:

The focus isn't as administrator. The task this is solving is pasting when ownership differs, which allows you to choose the ownership.

Here's a quick draft of what that could be like. Please make suggestions based on this, if you think it should be different:

```suggestion
## Files: Paste files with different ownership

Pasting copied files into a directory that has different ownership now shows a confirmation to either paste with the same ownership as the directory, or allow you to choose between pasting as an administrator or changing ownership to your own account.

Usually, having the same file ownership as the directory is the right choice. However, some circumstances may require having the ownership of files differ from the directory they are contained within.
```

WDYT?

Author: This really is about admin, though: if you're copy-pasting as a normal user you have no choice: you create the files as yourself.

---
title: Cockpit 333
author:
date: '2025-02-13'
tags: podman, files
slug: cockpit-333
category: release
summary: ''
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 333 🥃, cockpit-podman 101, and cockpit-files 16:

## Podman: Automatically start podman.socket

If the system or user's `podman.socket` unit isn't running, cockpit-podman now silently starts it on its own, instead of asking the user about it. There is no need to have the service enabled (i.e. always start it on boot or user login) for cockpit-podman any more.

> ## Files: Support copy&pasting files with administrator privileges

Reviewer: We could shorten this to:

```suggestion
## Files: Paste as administrator
```

...which is fine, IMO, or provide a little more context, as "copy and paste" is a commonly used phrase:

```suggestion
## Files: Copy and paste as an administrator
```

Which might be better, as it's clearer.
